lib/posix-user/exportsyms.uk

```diff
@@ -64,3 +64,6 @@ getgrnam_r
 setgrent
 endgrent
 getgrent
+getresgid
```

Comment:
@mogasergiu 
getresgid is also mentioned on line 58. Shouldn't it be deleted from there?